src/tracks.c

```diff
@@ -3561,15 +3561,15 @@ void func_8002EEEC(s32 arg0) {
     s32 var_v1;
     s32 tempIdx;
 
-    sp88[0] = gNewShadowObj->segment.trans.x_position + gNewShadowWidth;
-    sp88[1] = gNewShadowObj->segment.trans.z_position + gNewShadowLength;
-    sp88[2] = gNewShadowObj->segment.trans.x_position - gNewShadowWidth;
-    sp88[3] = gNewShadowObj->segment.trans.z_position + gNewShadowLength;
-    sp88[4] = gNewShadowObj->segment.trans.x_position - gNewShadowWidth;
-    sp88[5] = gNewShadowObj->segment.trans.z_position - gNewShadowLength;
-    sp88[6] = gNewShadowObj->segment.trans.x_position + gNewShadowWidth;
-    sp88[7] = gNewShadowObj->segment.trans.z_position - gNewShadowLength;
-    temp_v0 = func_800BDC80(arg0, D_8011C3B8, &D_8011C8B8[D_8011D0B8], sp88[4], sp88[5], sp88[0], sp88[1]);
+    sp88[0].f[0] = gNewShadowObj->segment.trans.x_position + gNewShadowWidth;
```

Comment:
is float.x/float.y possible over float.f[0]/float.f[1]?

Reply:
Good call, xy worked fine